module/Application/view/error/index.phtml

```diff
@@ -1,62 +1,58 @@
-<h1><?php echo $this->translate('An error occurred') ?></h1>
-<h2><?php echo $this->message ?></h2>
+<h1>An error occurred</h1>
+<h2><?= $this->message ?></h2>
 
-<?php if (isset($this->display_exceptions) && $this->display_exceptions): ?>
+<?php if (isset($this->display_exceptions) && $this->display_exceptions) : ?>
 
-<?php if(isset($this->exception) && $this->exception instanceof Exception): ?>
+    <?php if (isset($this->exception) && $this->exception instanceof Exception) : ?>
 <hr/>
-<h2><?php echo $this->translate('Additional information') ?>:</h2>
-<h3><?php echo get_class($this->exception); ?></h3>
+<h2>Additional information:</h2>
```

Comment:
Why this part is not with 2 indents? It is body of `if` statement, so I think we should have:
````
<?php if (something) : ?>
    <some html="tag">...</some>
<?php endif ?>
```

Of course the same for loops.